### PR TITLE
Use exec

### PR DESCRIPTION
**Pull request by erikh**
requires #178 #176

This implements functionality to execute a command under use lock.

### CODE REVIEW

File: volcli/volcli.go [hunk 1]
```diff
@@ -567,3 +572,73 @@ func useTheForce(ctx *cli.Context) (bool, error) {
...
+
+	args := ctx.Args()[1:]
+	if args[0] == "--" {
+		if len(args) < 2 {
+			return true, fmt.Errorf("You must supply a command to run")
+		}
+		args = args[1:]
+	}
+
+	err = lock.NewDriver(cfg).ExecuteWithMultiUseLock([]config.UseLocker{um, us}, true, 0, func(ld *lock.Driver, uls []config.UseLocker) error {
```
Comment (mapuri) [inline volcli/volcli.go]: just wondering, should be pick the timeout from `GlobalConfig` instead of fixing it to zero?

**Comment by mapuri** [comment]
LGTM, this can pretty nice debug utility to test lock issues I think.

File: volcli/volcli.go (hunk 1 above)
Comment (erikh) [inline volcli/volcli.go]: Not in this case. We want to hang as long as the user doesn't SIGINT us.

Comment (mapuri) [inline volcli/volcli.go]: ah, ok makes sense

**Pull request merged**